packages/xrpl/test/integration/transactions/oracleDelete.test.ts

```diff
@@ -0,0 +1,78 @@
+import { stringToHex } from '@xrplf/isomorphic/dist/utils'
+
+import { OracleSet, OracleDelete, RippledError } from '../../../src'
+import { assertRejects } from '../../testUtils'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('OracleDelete', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      const setTx: OracleSet = {
+        TransactionType: 'OracleSet',
+        Account: testContext.wallet.classicAddress,
+        OracleDocumentID: 1234,
+        LastUpdateTime: Math.floor(Date.now() / 1000),
+        PriceDataSeries: [
+          {
+            PriceData: {
+              BaseAsset: 'XRP',
+              QuoteAsset: 'USD',
+              AssetPrice: 740,
+              Scale: 3,
+            },
+          },
+        ],
+        Provider: stringToHex('chainlink'),
+        URI: '6469645F6578616D706C65',
+        AssetClass: stringToHex('currency'),
+      }
+
+      await testTransaction(testContext.client, setTx, testContext.wallet)
```

Comment:
The object should be verified to exist here.

Reply:
Bump - essentially I'd like to confirm that the object existed at all and something happened.

Reply:
Done in https://github.com/XRPLF/xrpl.js/pull/2688/commits/91695f7fc0b0d90c58063a357bc18829c955c244